packages/mdc-drag-drop/drag-collection/index.js

```diff
@@ -525,7 +536,18 @@ export class MDCDragCollection extends MDCComponent {
 
   handleDragMove_(e) {
     // (e.originalEvent || e.detail.originalEvent).preventDefault();
+    const scrollZone = MDCDragCollection.getScrollZone_(e, this.scrollZones_);
 
+    if(scrollZone) {
+      this.handleScrollZone_(scrollZone);
+      return;
```

Comment:
I can't decide if we should handle both scrolling _and_ drag-drop at the same time... Currently, when the user scrolls down the page while dragging, they have to stop scrolling and then move their finger a bit to trigger the drop zone to activate. On the other hand, if we removed this `return`, the user might drop a card somewhere accidentally... Maybe add a `TODO(moog16)` comment here with a note explaining that we should revisit whether to remove the `return` statement.

Reply:
Will do...yes I also wrestled with this. I switched the logic twice. The behavior I saw made me think that scrolling should overrule the dropzone because if the user activated a dropzone but is at very top/bottom page, the page wouldn't scroll. I did not try attempting both, since I thought it would be confusing to the user. And as you said would unknowingly drop a card.